recipes/labellib/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "LabelLib" %}
+{% set version = "2020.10.05" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Fluorescence-Tools/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 005a621ca69f0a0305867952aa641c48c579447c7fc0fdd1cc7e92a06c1f219a
+
+build:
+    number: 0
+    skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - eigen
+    - pybind11
+    - libcxx
+  host:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - LabelLib
+
+about:
+  home: https://github.com/Fluorescence-Tools/LabelLib
+  license: LGPL-2.1-or-later
+  license_family: LGPL
```

Comment:
I don't understand, the upstream repository says Mozilla Public License? And nothing resembling the license attached to this PR?